src/components/BookItem.js

```diff
@@ -1,22 +1,39 @@
-/* eslint-disable react/prop-types */
-import React from 'react';
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { getBooks, removeBook } from '../redux/books/books';
+
+function BookItem() {
+  const bookStore = useSelector((state) => state.booksReducer);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    if (!bookStore.length) {
+      dispatch(getBooks());
+    }
+  }, []);
```

Comment:
- [ ] Nice use of `useEffect` hook to make a server request. 👍 
- [ ] Highly organized code 💯